### PR TITLE
Cve 2019 14873

### PR DESCRIPTION
**Pull request by rehakm**

### CODE REVIEW

File: 2019/14xxx/CVE-2019-14872.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
+{
```
Comment (cve-team) [inline 2019/14xxx/CVE-2019-14872.json]: Invalid JSON file

File: 2019/14xxx/CVE-2019-14871.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            "name": "https://census-labs.com/news/2020/01/31/multiple-null-pointer-dereference-vulnerabilities-in-newlib/",
+            "refsource": "CONFIRM"
+        }
+    ]
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "The REENT_CHECK macro (see newlib/libc/include/sys/reent.h) as used by REENT_CHECK_TM, REENT_CHECK_MISC, REENT_CHECK_MP and other newlib macros in verisons prior to 3.3.0, does not check for memory allocation problems when the DEBUG flag is unset (as is the case in production firmware builds). "
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14871.json]: Minor typo, please `s/verisons/versions/`

Comment (rehakm) [reply 2019/14xxx/CVE-2019-14871.json]: corrected

**Comment by iamleot** [comment]
rehakm writes:
> [...]
> +        "value": "The REENT_CHECK macro (see newlib/libc/include/sys/reent.h) as used by REENT_CHECK_TM, REENT_CHECK_MISC, REENT_CHECK_MP and other newlib macros in verisons prior to 3.3.0, does not check for memory allocation problems when the DEBUG flag is unset (as is the case in production firmware builds). "
>
> corrected
> [...]

The .diff of this PR seems still with the typo.

**Comment by iamleot** [comment]
Leonardo Taccari writes:
> rehakm writes:
> > [...]
> > +        "value": "The REENT_CHECK macro (see newlib/libc/include/sys/reent.h) as used by REENT_CHECK_TM, REENT_CHECK_MISC, REENT_CHECK_MP and other newlib macros in verisons prior to 3.3.0, does not check for memory allocation problems when the DEBUG flag is unset (as is the case in production firmware builds). "
> >
> > corrected
> > [...]
>
> The .diff of this PR seems still with the typo.
>

...but this will be probably addressed by #3433 and/or #3434 so please
ignore my previous comments and thanks!

**Pull request merged**